nmostesting/suites/IS1101Test.py

```diff
@@ -303,585 +247,236 @@ def test_01_03(self, test):
+        if len(self.base_edid_inputs) == 0:
+            return test.UNCLEAR("Not tested. No inputs with Base EDID support found.")
+
+        for inputId in self.is11_utils.sampled_list(self.base_edid_inputs):
+            # Save the default value of the Effective EDID
+            default_edid = self.get_effective_edid(test, inputId)
+
+            # PUT the Base EDID to the Input
+            valid, response = self.do_request("PUT",
+                                              self.compat_url + "inputs/" + inputId + "/edid/base",
+                                              headers={"Content-Type": "application/octet-stream"},
+                                              data=VALID_EDID)
+            if not valid or response.status_code != 204:
+                return test.FAIL("Unexpected response from "
+                                 "the Stream Compatibility Management API: {}".format(response))
 
-            for input_id in self.not_edid_connected_inputs:
-                valid, response = TestHelper.do_request(
-                    "GET", self.compat_url + "inputs/" + input_id + "/edid/base/"
-                )
-                if not valid:
-                    return test.FAIL("Unexpected response from the streamcompatibility API: {}".format(response))
-                if response.status_code != 204:
-                    return test.FAIL("The input {} edid base streamcompatibility request has failed: {}"
-                                     .format(input_id, response))
-            return test.PASS()
+            # Verify that /edid/base returns the last Base EDID put
+            valid, response = self.do_request("GET", self.compat_url + "inputs/" + inputId + "/edid/base")
+            if (
+                not valid
+                or response.status_code != 200
+                or response.headers["Content-Type"] != "application/octet-stream"
+            ):
+                return test.FAIL("Unexpected response from "
+                                 "the Stream Compatibility Management API: {}".format(response))
+            if response.content != VALID_EDID:
+                return test.FAIL("The Base EDID of Input {}"
+                                 "doesn't match the Base EDID that has been put".format(inputId))
+
+            # Verify that /edid/effective returns the last Base EDID put
+            if self.get_effective_edid(test, inputId) != VALID_EDID:
+                return test.FAIL("The Effective EDID of Input {}"
+                                 "doesn't match the Base EDID that has been put".format(inputId))
+
+            # Delete the Base EDID
+            valid, response = self.do_request("DELETE", self.compat_url + "inputs/" + inputId + "/edid/base")
+            if not valid or response.status_code != 204:
+                return test.FAIL("Unexpected response from "
+                                 "the Stream Compatibility Management API: {}".format(response))
 
```

Comment:
Add reasonable delay after Delete command.
`start_time = time.time()`
`            while time.time() < start_time + CONFIG.API_PROCESSING_TIMEOUT:`
`                time.sleep(0.2)`

Reply:
The same case as https://github.com/AMWA-TV/nmos-testing/pull/836#discussion_r1373928073

Reply:
Fixed in https://github.com/AMWA-TV/nmos-testing/pull/836/commits/1f836762f1b45530a0ec681868882845bb3015dc